docs/validator/economics.md

```diff
@@ -5,71 +5,88 @@ sidebar_label: Understanding Economics
 
-### Receive third-party NEAR tokens via delegation
-
-NEAR Protocol provides *smart contracts* for delegators who want to stake $NEAR tokens without running a node. Therefore, every validator can implement its staking smart contract, or use the ones made available by NEAR Protocol or the community. 
+You can check current validators, upcoming validators and proposals via RPC:
+
+```bash
+$ http post https://rpc.betanet.nearprotocol.com jsonrpc=2.0 id=123 method=validators params:='[null]'
+{
+    ...
+    "result": {
+        "current_fisherman": ...
+        "current_proposals": ...
+        "current_validators": ...
+        "next_fisherman": ...
+        "next_validators": ...
+        "prev_epoch_kickout": []
+    }
+}
+```
+
+### Borrow NEAR tokens via stake delegation
+
+In NEAR Protocol token holders can lend their $NEAR to validators using *smart contracts* receiving percentage of reward validators capturing (also called delegation). Therefore, every validator can implement its staking smart contract, or use the ones made available by NEAR Protocol or the community. 
 This option is still work in progress, so please follow the [NEP on GitHub](https://github.com/nearprotocol/NEPs/blob/staking-contract/text/0000-staking-contract.md) for specifications and features discussion.
 
 <blockquote class="warning">
     <strong>heads up</strong><br><br>
-    In the beginning, the network will run on a limited number of shards, with all validators competing for a limited number of seats. As an example, with 40% of the total supply at stake (400 million out of 1 billion $NEAR tokens) and 100 seats available, every validator will need 4 million $NEAR tokens to take one seat.
+    In the beginning, the network will run on a limited number of shards, with all validators competing for a limited number of seats. As an example, with 40% of the total supply at stake (400 million out of 1 billion $NEAR) and 100 seats available, every validator will need 4 million $NEAR to take one seat.
 </blockquote>
 
 
-## 2. Nodes infrastructure
+## 2. Economic implications of running a node
 
-Once you obtained your stake in $NEAR, you need to setup the infrastructure to serve the network.
-At a high level, your nodes must provide:
+To actually operate the node and be able to become a validator, you need to setup the infrastructure.
+At a high level, your setup must provide:
 - Availability
 - Consistency
 
 ### Availability
 
-NEAR Protocol doesn't enforce slashing for downtime. However, validators failing to provide a minimum threshold of chunks and blocks within each epoch get *kicked out* and lose rewards.
+NEAR Protocol doesn't enforce slashing for downtime. However, validators failing to provide a minimum threshold of chunks and blocks within each epoch get *kicked out* and lose rewards for that epoch.
```

Comment:
In the blog post we adopted the definition "force unstaked" instead of "kicked out". I think it's more accurate